integration/pkg/tools/postinstall.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+echo "Creating a default user cwagent"
+COMMON_CONFIG_PATH=/opt/aws/amazon-cloudwatch-agent/etc/common-config.toml
+SAMPLE_SUFFIX=SAMPLE_DO_NOT_MODIFY
+if [ ! -f ${COMMON_CONFIG_PATH} ]; then
+    cp ${COMMON_CONFIG_PATH}.${SAMPLE_SUFFIX} ${COMMON_CONFIG_PATH}
+fi
+
+if [[ cwagent == `sudo dscl . -list /Users UniqueID | awk '{print $1}' | grep -w cwagent` ]]; then
+    echo "User already exists!"
+    exit 0
+fi
+
+LastID=`sudo dscl . -list /Users UniqueID | awk '{print $2}' | sort -n | tail -1`
+NextID=$((LastID + 1))
+
+. /etc/rc.common
+sudo dscl . create /Users/cwagent
+sudo dscl . create /Users/cwagent RealName cwagent
+
+sudo dscl . create /Users/cwagent UniqueID $NextID
+# PrimaryGroupID of 20 to create a standard user
+sudo dscl . create /Users/cwagent PrimaryGroupID 20
+sudo dscl . create /Users/cwagent UserShell /usr/bin/false
+sudo dscl . create /Users/cwagent NFSHomeDirectory /Users/cwagent
```

Comment:
Does this also need to create the aoc user?

Reply:
I'm not sure. I copied this code from the internal code. So I assume no but I have not tested.

Reply:
I will test this and report back.

Reply:
It seems aoc is not supported in darwin ctl. 
https://github.com/aws/amazon-cloudwatch-agent/blob/master/packaging/darwin/amazon-cloudwatch-agent-ctl#L36-L71 https://github.com/aws/amazon-cloudwatch-agent/blob/master/Makefile#L247-L259 not sure why this is not enabled